Confus/Confus/Health.cpp

```diff
@@ -0,0 +1,48 @@
+#include "Health.h"
+
+namespace Confus
+{
+	void Health::damage(int a_Damage)
+	{
+		if (a_Damage > 0)
+		{
+			m_Health -= a_Damage;
+			if (m_Health <= 0)
+			{
+				m_Health = 0;
+				m_DeathCallback(m_AttachedNode);
+			}
+			else if (m_Health <= 50)
+			{
```

Comment:
I don't see a reason for keeping this else statement here if it's not being used at all